DPG/Tasks/AOTTrack/qaEfficiency.cxx

```diff
@@ -817,14 +989,18 @@ struct QaEfficiency {
     LOG(info) << "Set doTr to: " << (doTr ? "true" : "false");
     LOG(info) << "Set doHe to: " << (doHe ? "true" : "false");
     LOG(info) << "Set doAl to: " << (doAl ? "true" : "false");
+    LOG(info) << "Set doAll to: " << (doAll ? "true" : "false");
     LOG(info) << "Set trackSelection to: " << (trackSelection ? "true" : "false");
     LOG(info) << "Set makeEff to: " << (makeEff ? "true" : "false");
     LOG(info) << "Set doPtEta to: " << (doPtEta ? "true" : "false");
 
+    PDGCfgs = {doEl.value, doMu.value, doPi.value, doKa.value, doPr.value, doDe.value, doTr.value, doHe.value, doAl.value};
+    LOG(info) << "Set PDGCfgs to: " << doEl.value << ", " << doMu.value << ", " << doPi.value << ", " << doKa.value << ", " << doPr.value << ", " << doDe.value << ", " << doTr.value << ", " << doHe.value << ", " << doAl.value;
+
     auto doLimits = [&](double& min, double& max, const ConfigurableAxis& binning) {
       const AxisSpec a{binning, "dummy"};
       min = a.binEdges[0];
-      max = a.binEdges[1];
+      max = a.binEdges[a.binEdges.size() - 1];
```

Comment:
Is this needed to work with variable width bins?

Reply:
this formula works with both fixed-width and variable-width bins. Specifically, for pt we might have variable width.